www/guides/react.md

```diff
@@ -0,0 +1,361 @@
+---
+layout: layouts/guide.njk
+permalink: '/guides/react/'
+title: The Snowpack Guide to Getting Started with React
+description: 'Get started with this in-depth tutorial on how to build React applications and websites with Snowpack. '
+date: 2020-10-01
+---
+
+Image: ideally a nice header image showing the before or after, or some illustration with elements of the Snowpack logo, React, and symbols related to speed, joy.
+
+Snowpack is a great fit for [React](https://reactjs.org/) projects of any size. It's easy to get started with and can scale to projects containing thousands of components and pages without any impact on development speed. Unlike traditional React application tooling, Snowpack saves you from getting bogged down with complex bundler setups and configuration files.
+
+In this guide, you'll go from an empty directory to a fully configured Snowpack project with support for React and several other useful optimizations. In the process, you'll learn:
+
+- How to set up Snowpack
+- How to get started with React in a Snowpack project
+- Importing CSS, images and other assets
+- Enabling [Fast Refresh](https://reactnative.dev/docs/fast-refresh) mode for React
+- Connecting your favorite developer tools to Snowpack
+
+> 💡 Tip: if you want to jump to the end to see a full featured React setup, the [Create Snowpack App React template](https://github.com/pikapkg/snowpack/tree/master/create-snowpack-app/app-template-react) comes with everything you'll learn in this tutorial plus other useful tools.
+>
+> ```bash
+> npx create-snowpack-project react-snowpack-template --template @snowpack/project-template-react
+> ```
+
+Prerequisites: Snowpack is a command line tool installed from npm. This guide assumes a basic understanding of Node.js, npm, and how to run commands in the terminal. Knowledge of React is not required, Snowpack is a great way to learn React!
+
+## Getting started with Snowpack and React
+
+The easiest way to start a new Snowpack project is with [Create Snowpack App](<https://www.snowpack.dev/#create-snowpack-app-(csa)>). Create Snowpack App is a tool to quickly set up Snowpack in a new directory.
+
+The template you're starting with here is `@snowpack/project-template-minimal` is a bare-bones setup where Snowpack serves an `index.html`, `index.css`, and `index.js`.
+
+Open up your terminal and head to a directory where you want to put your new project. Now run `npx create-snowpack-project react-snowpack --template @snowpack/project-template-minimal`. This creates a new directory called `react-snowpack` and installs the `@snowpack/project-template-minimal` inside it.
+
+You can now head to that new `react-snowpack` directory with `cd react-snowpack`. And to start up Snowpack run `npm run start`. You should see your new website up and running!
```

Comment:
Here and above: all commands that the user should run should look the same, stylistically. That means moving 

```
npx create-snowpack-project react-snowpack --template @snowpack/project-template-minimal
```

and 

```
cd react-snowpack
npm run start
```

into their own full code blocks. 

The text you have don't necessarily have to change,  although the create-snowpack-app command is long enough that you could probably replace that text with:

> Now run the following command in your terminal to create a new directory called `react-snowpack` that will automatically be filled with the minimal template & Snowpack setup.